storm-core/src/jvm/backtype/storm/scheduler/TopologyDetails.java

```diff
@@ -38,15 +39,19 @@
 
 
 public class TopologyDetails {
-    String topologyId;
-    Map topologyConf;
-    StormTopology topology;
-    Map<ExecutorDetails, String> executorToComponent;
-    int numWorkers;
+    private String topologyId;
+    private Map topologyConf;
+    private StormTopology topology;
+    private Map<ExecutorDetails, String> executorToComponent;
+    private int numWorkers;
```

Comment:
If we are making these private, it would be good to either prefix with `_` or use `this.` where they are referenced.

Do they need to be private?